src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -51,6 +51,18 @@ public WordUtils() {
       super();
     }
 
+    /* All common whitespaces as defined in Character.isWhitespace(char) */
+    private static final char[] WHITESPACE_CHARS =
+        { '\t', '\n', '\u000B', '\f', '\r', '\u001C', '\u001D', '\u001E', '\u001F', ' ' };
```

Comment:
This seems like it will be a maintenance headache. Can the stock method `Character.isWhitespace()` be used?

Reply:
The other way I can think of would be to rewrite parts of `capitalize(String,char...)` to reinclude it, but that would mean losing the performance gain whenever a delimiter isn't specified. If that's fine, I can go that way.